script/generateUtilsDoc.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+# Copyright (C) 2019 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+cd $TRAVIS_BUILD_DIR
+
+touch docs/utils.html
```

Comment:
@DrMullings Let's use this as a stepping stone, but we should have an include list instead... Maybe we can discuss this further in upcomming refinements or directly in some ticket